### ISSUE TITLE
Implement background subtraction in create_meta_pixels

### ISSUE THREAD

**Issue by paolomassa** [TCDSolar/stixpy#91]
I think that it is necessary to implement background subtraction at the level of pixel counts. In my opinion, the best place to have it implemented is inside `create_meta_pixels`. We could have add a keyword `bkg_cpd` containing a compressed pixel data structure for a bkg measurement. If we do not pass  `bkg_cpd` as input, then we do not perfom background subtraction

**Comment by samaloney** [comment]
Agreed we absolutely need to facilitate this but I'm not sure if `create_meta_pixels` is the best place.
